### DISCUSSION TITLE
Keys are shown behind modals

### DISCUSSION THREAD

**Discussion by amirHossein5** [tridactyl/tridactyl#4953] [General]
![Screenshot from 2024-03-25 09-47-24](https://github.com/tridactyl/tridactyl/assets/68776630/9272585a-a3a3-4862-a8eb-a1a01decd62e)

![Screenshot from 2024-03-25 11-33-22](https://github.com/tridactyl/tridactyl/assets/68776630/e03cafb5-20cf-43bb-9462-7fd516981938)

[https://monkeytype.com](https://monkeytype.com)

![Screenshot from 2024-03-25 12-05-24](https://github.com/tridactyl/tridactyl/assets/68776630/4a8eb25c-4772-4bf7-b71c-a2fdbf1ab357)

**Comment by bovine3dom** [comment]
Thanks for reporting, I've moved this to an issue because it's probably a bug we can fix or at least think about fixing :)

https://github.com/tridactyl/tridactyl/issues/4954
